addons/http_routing/models/ir_http.py

```diff
@@ -256,6 +256,8 @@ def is_a_bot(cls):
     @classmethod
     def get_nearest_lang(cls, lang):
         # Try to find a similar lang. Eg: fr_BE and fr_FR
+        if lang in request.env['ir.module.module']._installed().keys():
```

Comment:
`.keys()` is actually not needed, `in` on a `dict` will use keys on keys

Reply:
Indeed, good catch, done